quickcheck-dynamic-iosim/test/Spec/DynamicLogic/RegistryModel.hs

```diff
@@ -48,12 +50,12 @@ deriving instance Eq (Action RegState a)
 
 instance StateModel RegState where
   data Action RegState a where
-    Init :: Action RegState ()
-    Spawn :: Action RegState ModelThreadId
-    WhereIs :: String -> Action RegState (Maybe ModelThreadId)
-    Register :: String -> Var ModelThreadId -> Action RegState (Either SomeException ())
+    Init       :: Action RegState ()
```

Comment:
nitpick: We should settle on a single code formatter in order to avoid these kind of changes. The default provided by the nix-shell is fourmolu with some configuration. I am fine with any other formatter so long as there's only one. We should probably put a "coding standards" somewhere too.

Reply:
Petition to make a separate PR right after this one that fixes formatting and introduces a `formolu` standard? Perhaps we should also put it in CI to be sure?

Reply:
I ran `fourmolu` in `nix-shell` (without any input - how works?!) and nothing happened... I then ran:
```
find . -type f -name *.hs | xargs -n1 fourmolu
```
and it just spat out a bunch of stuff on `stdout`. Is there some unknown magic I need to run to make it actually format every file in place?

Reply:
Yeah, it needs an additional argument. I agree we should leave it as it is now and add something more automated later :)

Reply:
Do you know what the additional argument is? ;)

Reply:
`-i` (like _in-place_, the long version is `--mode inplace`)